src/pages/HouseRule.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+import Icon from '@mdi/react';
+import { mdiMenuUp, mdiMenuRight } from '@mdi/js';
+import LineDivider from '../components/LineDivider';
+
+interface RuleProps {
+  id: number;
+  title: string;
+  content: string | React.ReactNode;
+}
+
+const Rule = ({ id, title, content }: RuleProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleRule = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <section className="rule-container">
+      <div className="rule-content">
+        <h5
+          style={{
+            color: isOpen ? '#ff6700' : '',
+          }}
+        >
+          {id}. {title}
+        </h5>
+        <button
+          className={`active-btn ${isOpen ? 'toggled' : ''}`}
+          onClick={toggleRule}
+        >
+          <Icon path={isOpen ? mdiMenuUp : mdiMenuRight} className="mdi-icon" />
+        </button>
+      </div>
+      {isOpen && (
+        <>
+          <LineDivider /> {content}
```

Comment:
https://react.dev/reference/react/Children

```suggestion
          <LineDivider /> {children}
```